testing/nand_sim/nand_sim_raw.c

```diff
@@ -0,0 +1,250 @@
+/****************************************************************************
+ * apps/testing/nand_sim/nand_sim_raw.c
```

Comment:
nand_sim_raw is a nand raw lowhalf driver, so it should be moved to.nuttx/drivers/mtd